File: C/2.Intermediate/sum of n numbers without for loop.c
#include<stdio.h>
#include<conio.h>

int main(){
   clrscr();
   int number,sum;
   printf("Enter the end number till which you want the sum of eg- if you enter 5 then(1+2+3+4+5)=15.\n");
   scanf("%d", &number);
   sum = number * (number + 1)/2;
   printf("The sum is: %d\n", sum); 
   return 0;
}

File: Java/2.Intermediate/sum of n numbers without for loop.java
import java.util.*;
import java.lang.*;

class Sum // Please put the classname as the specified filename to solve the error
{  
    public static void main(String args[])
    {
        int number,i;
        int sum = 0;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a number till which you want a sum of!");
        number = scanner.nextInt();
        sum = number * (number + 1)/2;
        System.out.println("The sum is: "+sum);
    }
}